src/ui/pages/Overview/Positions/EmptyPositionsView.tsx

```diff
@@ -76,58 +175,53 @@ export function EmptyPositionsView() {
     return <EmptyView>No assets yet</EmptyView>;
   }
 
+  const buyCryptoHref = `${ZERION_ORIGIN}/deposit?${addWalletParams}`;
+
   return (
-    <VStack
-      gap={16}
-      style={{
-        justifyItems: 'stretch',
-        paddingInline: 16,
-        textAlign: 'center',
-      }}
-    >
-      <VStack gap={12} style={{ justifyItems: 'center' }}>
-        <img
-          alt="Empty Wallet Cover"
-          src="https://cdn.zerion.io/images/dna-assets/empty-wallet-img.png"
-          srcSet="https://cdn.zerion.io/images/dna-assets/empty-wallet-img.png, https://cdn.zerion.io/images/dna-assets/empty-wallet-img_2x.png 2x"
-          style={{ height: 64 }}
-        />
-        <VStack gap={0}>
-          <UIText kind="headline/h3">Fund your wallet</UIText>
-          <UIText kind="body/regular" color="var(--neutral-600)">
-            Buy or transfer crypto to get started
-          </UIText>
+    <>
+      <VStack
+        gap={16}
+        style={{
+          justifyItems: 'stretch',
+          paddingInline: 16,
+          textAlign: 'center',
+          marginTop: 32,
+        }}
+      >
+        <VStack gap={12} style={{ justifyItems: 'center' }}>
+          <img
+            alt=""
+            src="https://cdn.zerion.io/images/dna-assets/empty-wallet-img.png"
+            srcSet="https://cdn.zerion.io/images/dna-assets/empty-wallet-img.png, https://cdn.zerion.io/images/dna-assets/empty-wallet-img_2x.png 2x"
+            style={{ height: 64 }}
+          />
+          <VStack gap={0}>
+            <UIText kind="headline/h1">Get Started</UIText>
+            <UIText kind="body/regular" color="var(--neutral-600)">
+              By adding crypto to your wallet
+            </UIText>
+          </VStack>
         </VStack>
-      </VStack>
-      <VStack gap={8}>
-        <Button
-          size={48}
-          kind="primary"
-          as={UnstyledAnchor}
-          href={`${ZERION_ORIGIN}/deposit?${addWalletParams}`}
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Buy Crypto with Card
-        </Button>
-        <Button
-          size={44}
-          kind="regular"
-          as={UnstyledLink}
-          to={`/receive?address=${wallet.address}`}
-        >
-          Receive from Another Wallet
-        </Button>
-        {getAddressType(wallet.address) === 'solana' ? (
+        <VStack gap={8}>
           <Button
-            kind="regular"
-            size={44}
-            onClick={() => goToBridgeMutation.mutate()}
+            size={48}
+            kind="primary"
+            onClick={() => {
+              dialogRef.current?.showModal();
+            }}
           >
-            Transfer from Ethereum
+            <HStack gap={8} alignItems="center" justifyContent="center">
+              <ArrowCircleIcon />
+              <span>Fund</span>
```

Comment:
Is this the final copy? Looks too short :))

Reply:
Seems so!